starter-page-templates.php

```diff
@@ -1,12 +1,11 @@
 <?php
 /**
- * Plugin Name: WP JS Plugin Starter
- * Plugin URI: https://github.com/youknowriad/wp-js-plugin-starter
- * Description: Just another WordPress plugin starter
- * Version: 1.0.1
- * Author: Riad Benguella
+ * Plugin Name: Starter Page Templates
+ * Description: Create new pages by selecting pre-built page templates.
+ * Version: 1.0.0
+ * Author: Automattic
  *
- * @package wp-js-plugin-starter
+ * @package starter-page-templates
```

Comment:
Question - should our package strings really be `a8c` namespaced to avoid clashes with other packages? Applies to `domain` argument for i18n also.

I'm not sure of the answer...

Reply:
good question here! I also don't have the answer though :D

Reply:
We shouldn't have any problems as long as package names correspond to their slug in the plugin directory